src/main/webapp/app/course/manage/course-management.service.ts

```
@@ -703,4 +711,56 @@
+
+                    if (prefixA === prefixB) {
+                        return 0; // Both semesters are the same (either both WS or both SS)
+                    }
+
+                    return prefixA === 'WS' ? -1 : 1; // WS should be placed above SS
+                })
+        );
+    }
+
+    getSemesterCollapseStateFromStorage(storageId: string): boolean {
+        const storedCollapseState: string | null = localStorage.getItem('semester.collapseState.' + storageId);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)